mitodl/mitxonline
# add backfill migration for new table paid courserun

#### Pre-Flight checklist


#### What are the relevant tickets?
(https://github.com/mitodl/mitxonline/issues/738)

#### What's this PR do?
To backfill the new table paid_course_runs for existing orders

#### How should this be manually tested?
-  Run ```docker-compose run --rm  web ./manage.py migrate``` locally
-  Go to /admin/courses/paidcourserun/, all the paid courserun orders should be populated there.
-  Verify the new records with admin/ecommerce/order/
